src/preprocessing/readvtk/vtk2trixi.jl

```diff
@@ -0,0 +1,49 @@
+"""
+    vtk2trixi(file::String)
+
+Convert data from VTK-file to InitialCondition 
+
+# Arguments
+- `file`: Name of the file to be loaded. 
+"""
+function vtk2trixi(file)
+    vtk_file = VTKFile(file)
+
+    # Retrieve data fields (e.g., pressure, velocity, ...)
+    point_data = get_point_data(vtk_file)
```

Comment:
```suggestion
    point_data = ReadVTK.get_point_data(vtk_file)
```
Please adapt this for the other functions as well. It's only used in this scope so far. So it's better not to export these functions.